CharacterCreation/02_Classes.txt

```diff
@@ -190,7 +203,8 @@ Class Primary Attribute: Intelligence
 Starting Wealth: $2580 + 100*Luck
 
 passives: 
-* Gain Skill [Repair] at 10. 
+* Gain Skill Repair at 10.
+* Gain Skill Jury-rigging at 10.
```

Comment:
Why the buff to Engineer?

Reply:
We can drop it, but I like the concept of giving Jury-rigging to engineers. Also, I generally wanted to give each class about 20 skill points points passive.

Reply:
Also worth noting that Opossum had originally folded Jury-rig into Repair when he put up the original skills update doc. Now that it's become its own skill again, not adding it would have been a nerf.